mypy/main.py

```diff
@@ -31,6 +33,40 @@ def __init__(self) -> None:
         self.dirty_stubs = False
 
 
+class MyPyCommand(setuptools.Command):
+    """The :class:`MyPyCommand` class is used by setuptools to perform
+    type checks on registered modules.
+    """
+
+    description = "Validate PEP 0484 Types with mypy"
+    user_options = [
+        ('mypy-args=', None, 'Arguments to pass through to mypy')
+    ]  # type: List[tuple]
+
+    def initialize_options(self):
+        self.mypy_args = ''
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        try:
+            mypy_options = []
+            for package in self.distribution.packages:
```

Comment:
Switched this to use a set, most package authors use tools like `find_packages` which I don't believe will return dupes, but we can handle them by switching to a set so we might as well.